src/Testing/TypeInferenceTestCase.php

```diff
@@ -398,7 +398,7 @@ private static function isFileLintSkipped(string $file): bool
 
 			@fclose($f);
 
-			if (preg_match('~<?php\\s*\\/\\/\s*lint\s*([^\d\s]+)\s*([^\s]+)\s*~i', $firstLine, $m) === 1) {
+			if (preg_match('~<?php\\s*(?:declare\\s*\([^)]+\)\\s*;\\s*)?\\/\\/\s*lint\s*([^\d\s]+)\s*([^\s]+)\s*~i', $firstLine, $m) === 1) {
```

Comment:
added support for `<?php declare(strict_types = 0); // lint >= 8.1` in `gatherAssertTypesFromDirectory`